requirements.txt

```diff
@@ -0,0 +1 @@
+pycryptodome==3.9.0
```

Comment:
Should structlog be added here?

Reply:
It should yes, please disregard this pull request, I'm just working on the branch.